covbot.py

```diff
@@ -609,7 +609,7 @@ def _short_location(self, location: str, length=int(12)) -> str:
         intensive_care_rate = (1 / math.pi) + -0.358209 / (1 + 0.0827213 * math.exp(0.035025 * age))
         hospitalization_rate = -0.0353956 - age * -0.00576217
         survival_rate = 1 - death_rate
-        pf = lambda x: round(x * 100)
+        pf = lambda x: round(x * 100, 1)
```

Comment:
You can format directly as percentage and omit the lambda, eg `{:.1%}` will render 0.3333333333 as `33.3%`

Reply:
ohh nice didnt knew this trick ! i am doing this now